### PR TITLE
Add support for dotnet-sdk8-preview

### PR DESCRIPTION
**Pull request by mscottford**
Adds support for the latest SDK 8 version. Hopefully, I followed the instructions in the contributor's guide correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ dotnet --list-sdks
 
 | Version             | DotNet SDK     | Arch        | Remarks
 |---------------------|----------------|-------------|---------
+| `dotnet-sdk8-preview` | dotnet 8.0.100-preview.6 | x64 & arm64 |
```
Comment (isen-ng) [inline README.md:26]: Please move this line to the `Preview versions` section instead

**Comment by isen-ng** [comment]
Thank you for your contribution! There is 1 slight comment ..

**Comment by mscottford** [comment]
@isen-ng All done. Please let me know if you want me to address anything else.

**Comment by isen-ng** [comment]
Awesome @mscottford! Thank you for your contribution!

**Pull request merged**